mmcv/ops/prroi_pool.py

```diff
@@ -32,11 +32,13 @@ def forward(ctx,
                 rois: torch.Tensor,
                 output_size: Tuple,
                 spatial_scale: float = 1.0) -> torch.Tensor:
-        if 'FloatTensor' not in features.type(
-        ) or 'FloatTensor' not in rois.type():
-            raise ValueError(
-                'Precise RoI Pooling only takes float input, got '
-                f'{features.type()} for features and {rois.type()} for rois.')
+        if torch.__version__ != 'parrots':
```

Comment:
Why does parrots skip the type checking?

Reply:
parrots tensor.type is not iterable, and implementation is a bit different from pytorch. have added judging logic for safety.